roocs_grids/scripts/create_grid_file.py

```diff
@@ -30,6 +33,7 @@
 # Create clore.Grid object
 g = clore.Grid(ds)
 # Drop variables and global attributes
+# FIXME: This is broken in newer versions of xarray
```

Comment:
I will need to have a look in the clisops regridding PR when time allows, to solve this.